selfdrive/monitoring/driver_monitor.py

```diff
@@ -27,19 +27,19 @@ def __init__(self, TICI=TICI, DT_DMON=DT_DMON):
 
     self._FACE_THRESHOLD = 0.5
     self._PARTIAL_FACE_THRESHOLD = 0.765 if TICI else 0.455
-    self._EYE_THRESHOLD = 0.25 if TICI else 0.57
-    self._SG_THRESHOLD = 0.83
-    self._BLINK_THRESHOLD = 0.62 if TICI else 0.68
-    self._BLINK_THRESHOLD_SLACK = 0.82 if TICI else 0.88
-    self._BLINK_THRESHOLD_STRICT = 0.62 if TICI else 0.68
+    self._EYE_THRESHOLD = 0.75 if TICI else 0.55
```

Comment:
This seems like a huge change, any particular cause for this?

Reply:
the old dataset only had like ~15 people and had a fair amount of sunglasses data that didn't get filtered out.
this is fixed in new one (which looks closer to c2 distribution)